packages/legend-server-depot/src/models/VersionedProjectData.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createModelSchema, primitive } from 'serializr';
+import { SerializationFactory } from '@finos/legend-shared';
+
+export class VersionedProjectData {
```

Comment:
isnt this supposed to be called `StoreProjectVersionData` instead?

Reply:
Not really. StoreProjectData would just have infor about groupId, artifactId, versionId but ProjectData would have info about dependecies and properties as well. This particular class has the ProjectData info of each version so i called it VersionedProjectData